app/controllers/users_controller.rb

```diff
@@ -8,6 +8,22 @@ def show
       @location = Location.find(@user.location_id)
     end
     @favorite_discussions = @user.favorite_discussions
+    if @user.is_admin
```

Comment:
you need to define @user somewhere before you use it. Try passing in the admin's admin_id as another param when you make the request and then do User.find(params[:admin_id])